docs/FRSKY_R9.md

```diff
@@ -21,31 +21,49 @@ The Frsky R9M transmitter module and R9MX receiver are commercially available an
+There are several DIY approaches for building an inverter dongle. A common approach is based on the MAX3232 RS232 chip; an excellent build tutorial is provided here [Some soldering required](https://discuss.ardupilot.org/t/some-soldering-required/27613). Be aware however that many fake chips are available, and the original scheme suggested in this blog may not work. One, in fact, may have to add an extra diode and resistor.
 
 <img src="images/frsky-max3232-inverter-scheme.jpg" width="360px">
 
+### ESP32 WiFi Bridge ###
+
+The mLRS git repository includes an Arduino sketch which allows several supported ESP32 boards to be used as a WiFi Bridge to connect the serial port to any of the many available GCS such as Mission Planner or QGroundControl.  This approach can also eliminate the need for a separate inverter circuit.  Two of these boards, the M5Stamp Pico Mate and the M5Stack M5Stamp C3U Mate allow pin layouts which are especially convenient to connect directly to the serial pins on the back of the R9M Tx module.  The [M5Stack M5Stamp C3U Mate](https://shop.m5stack.com/collections/m5-controllers/products/m5stamp-c3u-mate-with-pin-headers) is the easiest option as it can be flashed via its included USB port rather than requiring a separate programmer.  The 2.4 GHz WiFi bridge works especially well with 900 MHz radios like the R9M since the separate frequency range minimizes interference.
+
+To install the the sketch on the M5Stamp C3U, use the Arduino IDE.  Open the mlrs-wifi-bridge.ino sketch from the mLRS esp folder, edit the mlrs-wifi-bridge.ino file and uncomment just the MODULE\_M5STAMP\_C3U\_MATE\_FOR\_FRSKY\_R9M define, select the ESP32C3 Dev board in the IDE, connect the M5Stamp C3U module USB connector to your computer while holding down the center button, and upload the sketch via the IDE.
+
+__Be sure to unplug the M5Stamp C3U from the back of the R9M when programming via USB to avoid feeding 5 volt power back to R9M which might cause damage.__
+
+<img src="images/M5Stamp_C3U_installed.jpg" width="360px">
+
+Connecting the M5Stamp C3U to the R9M is easy:  Remove the screw and the plastic cover.  Cut a 5 pin length of the included pin header female connector and pull out the pin from the last position.  This position serves a key to avoid plugging in the board incorrectly.  Solder the pins in the thru holes as shown below (key position hanging over the left) and reinstall the plastic cover.  After programming the sketch, install on the back of the R9M as shown above.  Then, you can connect your GCS computer or mobile device to the mLRS_AP WiFi access point and connect the GCS via UDP on port 14550.
```

Comment:
- is it "... serves as key ..."?
- I guess whould be " ... "mLRS AP UDP" Wifi .."

Reply:
I'm not sure what your second comment means.  It's two steps, first connect the computer via WiFi to the access point, then connect the GCS via UDP

Reply:
it refers to that the wifi ap will show up in the list as "mLRS AP UDP"